flask/app/routes.py

```diff
@@ -506,3 +506,56 @@ def delete_tissue(id: int):
         return "Tissue has dataset(s), cannot delete", 422
     else:
         return "Not Found", 404
+
+@app.route("/api/tissue_samples", methods=["POST"])
+@login_required
+def post_tissue():
+    if not request.json:
+        return "Request body must be JSON", 400
+
+    models.Participant.query.filter_by(participant_id=request.json["participant_id"]).first_or_404()
+
+    enum_error = enum_validate(
+        models.TissueSample,
+        request.json,
+        ["tissue_sample_type","tissue_processing"]
+        )
+
+    if enum_error:
+        return enum_error, 400
+
+    now = datetime.now()
+    created = now
+    updated = now
```

Comment:
These are automatically set by SQLAlchemy because of how we wrote `models.py` so you don't need to set them.